src/components/QuranReader/index.tsx

```diff
@@ -75,7 +75,7 @@ const QuranReader = ({
     {
       fallbackData:
         isUsingDefaultTranslations && isUsingDefaultTafsirs && isUsingDefaultReciter
-          ? initialData.verses
+          ? [initialData.verses]
```

Comment:
Why did we do this change?

Reply:
ah, yeah. There was some typing issue, discussed this with @osamasayed 

For now we are reverting back to `initialData.verses`. But there's some issue our current `next.quran.com`. It still fetching the data on the client, even though we already specific 'fallbackData'. 

We can discuss this tomorrow and fix it in another PR